pulsar-common/src/test/java/org/apache/pulsar/common/util/collections/GrowableArrayBlockingQueueTest.java

```diff
@@ -131,11 +132,8 @@ public void blockingTake() throws Exception {
                 queue.put(n);
                 ++n;
             }
-
             // Wait until all the entries are consumed
-            while (!queue.isEmpty()) {
-                Thread.sleep(1);
-            }
+            Awaitility.await().until(queue::isEmpty);
```

Comment:
Awaitility will not pool every millisecond by default
this change will slow down the test and probably is is overwhelming
this is a test about a BlockingQueue, it is fine to have a Thread.sleep here
it is not a cause of flaky tests

Reply:
I already fix it.